docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -1332,9 +1332,9 @@ read_circuit_breaker:
   # CLI flag: -ingester.read-circuit-breaker.cooldown-period
   [cooldown_period: <duration> | default = 10s]
 
-  # (experimental) How long the circuit breaker should wait between an
-  # activation request and becoming effectively active. During that time both
-  # failures and successes will not be counted.
+  # (experimental) How long after an initial request an activated circuit
+  # breaker should wait before becoming effectively active. During this time
+  # neither failures nor successes will not be counted.
   # CLI flag: -ingester.read-circuit-breaker.initial-delay
```

Comment:
Same suggestion as above here.

Reply:
addressed in https://github.com/grafana/mimir/pull/9842/commits/24a3b0db894298d97e53aab419553c6d71074ec9